test/bridge.test.ts

```diff
@@ -28,8 +29,8 @@ chai.use(solidity);
 const MAX_UINT256 = hre.ethers.constants.MaxInt256;
 
 // Amount of dai and usdc to transfer to the user. Issued twice for aDai and aUsdc
-const DAI_UNIT = 1000000n * BigInt(10 ** 18);
-const USDC_UNIT = 1000000n * BigInt(10 ** 6);
+const DAI_UNIT = 1000n * BigInt(10 ** 18);
```

Comment:
Why do we multiply the DAI & USDC units by `1000n`? We should be increasing the `daiAmount ` & `usdcAmount` instead